airflow-core/tests/unit/models/test_dag.py

```diff
@@ -1187,6 +1188,24 @@ def test_create_dagrun_job_id_is_set(self, testing_dag_bundle):
         )
         assert dr.creating_job_id == job_id
 
+    @pytest.mark.parametrize(["partition_key"], [[None], ["my-key"], [123]])
+    def test_create_dagrun_partition_key(self, partition_key, testing_dag_bundle):
+        dag = DAG(dag_id="test_create_dagrun_partition_key", schedule=None)
+        scheduler_dag = sync_dag_to_db(dag)
```

Comment:
```suggestion
    def test_create_dagrun_partition_key(self, partition_key, dag_maker):
        with dag_maker("test_create_dagrun_partition_key") as schedular_dag:
```
nit: this should work as well, eliminating testing_dag_bunlde and sync_dag_to_db steps

Reply:
updated to use dag_maker